src/const.ts

```diff
@@ -1,303 +1,303 @@
-import { toByteArray } from "./util";
```

Comment:
You've changed the new lines of this file, please revert.

Reply:
Reverted - apologies